h2o-docs/src/product/data-science/glm.rst

```diff
@@ -1740,7 +1741,41 @@ Regress :math:`z_{i}` on the predictors :math:`x_{i}` using the weights :math:`w
 
 This process is repeated until the estimates :math:`\hat{\beta}` change by less than the specified amount.
 
-**Cost of computation**
+**Tweedie Variance Power Maximum Likelihood**
```

Comment:
```suggestion
**Tweedie Likelihood Calculation**
```
I think @syzonyuliia's documentation about Likelihood and AIC should refer here for the Tweedie likelihood calculation.